Dev/RescueTime/rescuetime.1h.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env PYTHONIOENCODING=UTF-8 # /usr/local/bin/python3
```

Comment:
This shebang means the script won't run correctly..  I believe you might be looking foir:

```
#! /usr/bin/env PYTHONIOENCODING=UTF-8 python3
```